include/users/modules/class-bulk-delete-users-by-user-role.php

```diff
@@ -103,7 +103,12 @@ public function render() {
 	 */
 	public function process() {
 		$delete_options                   = array();
-		$delete_options['selected_roles'] = array_get( $_POST, 'smbd_u_roles' );
+
+		/*
```

Comment:
In future use single line comments for this, not multi-line comments.

Reserve multi-line comments for PHPDocs